app/helpers/user_services_helper.rb

```diff
@@ -1,5 +1,84 @@
+# def get_create_user_params
+#   @input = parse_request
+#   print @input
+#   validates_presence_of :email_id, :password, :gender, :address
+# end
+
+def get_create_address_params
+  @input = parse_request
+  validates_presence_of :name, :line1, :line2, :city, :state, :pincode , :phone_number
+end
+
+def create_address(address_params, user_id)
```

Comment:
Object creator method should not be in helpers. Please move this to Address model